theano/gpuarray/opt.py

```diff
@@ -976,11 +981,10 @@ def local_gpua_advanced_incsubtensor(op, context_name, inputs):
 @register_opt('fast_compile')
 @op_lifter([tensor.CAReduce, tensor.Sum, tensor.elemwise.Prod])
 @register_opt2([tensor.CAReduce, tensor.Sum, tensor.elemwise.Prod], 'fast_compile')
-def local_gpua_careduce(op, context_name, inputs):
+def local_gpua_careduce(op, context_name, inputs, outputs):
     if isinstance(op.scalar_op, (scalar.Add, scalar.Mul,
                                       scalar.Maximum, scalar.Minimum)):
 
-        node = op.make_node(*inputs)
```

Comment:
When i was going through the profiling result per node, I saw that this takes the considerably longer time and thought creating the node would be the reason and passed the outputs as the parameter. But the time taken is almost the same with very negligible time difference. @nouiz If you think this is a good idea, i'd keep it